### ISSUE TITLE
TypeScript type for react-jsx-highmaps

### ISSUE THREAD

**Issue by sai109** [whawker/react-jsx-highcharts#343]
#### Bug or Feature Request?
Feature request

#### Description
The library is really nice to use, but there are no TypeScript type definitions for react-jsx-highmaps. I was wondering if some can be added as it would be helpful to anyone who uses the library with TypeScript.

#### How to reproduce
Install the react-jsx-highmaps and you get a tslint error complaining about the lack of type definitions, it can be fixed with an @ts-ignore.

#### Versions
* React JSX Highcharts version: 4.2.1
* Highcharts version: 9.0.1
* React version: 17.0.2
* React JSX Highmaps version: 2.2.2

**Comment by AspieCoder1** [comment]
Thanks, I have started having a look at implementing something similar for react-jsx-highcharts.

**Comment by AspieCoder1** [comment]
I have opened #344 which adds the type definitions for react-jsx-highmaps.

**Comment by anajavi** [comment]
@AspieCoder1 this is now released as react-jsx-highmaps@2.3.0-alpha.0, can you verify that it works for you?

**Comment by AspieCoder1** [comment]
Yes, I am able to download it and it appears the type definitions are working.

**Comment by anajavi** [comment]
Released as react-jsx-highmaps@2.3.0. Thanks for the PR!
